.github/workflows/ci.yaml

```diff
@@ -30,11 +30,13 @@ jobs:
           yarn test:all:no-watch
           yarn tsc
           yarn build:all
-          yarn changeset status --since=origin/main
 
   publish:
     if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
@realshuting is it possible to get a token with these permissions? I'm not sure if the Kyverno org allows Github actions to create and approve pull requests.